api/services/userService.js

```diff
@@ -0,0 +1,71 @@
+import jwt from 'jsonwebtoken';
+import axios from 'axios';
+
+import userDao from '../models/index.js';
+
+const kakaoLogin = async (kakaoToken) => {
+  const result = await axios({
+    method: 'GET',
+    url: 'https://kapi.kakao.com/v2/user/me',
+    headers: {
+      Authorization: `Bearer ${kakaoToken}`,
+      'Content-Type': 'application/x-www-form-urlencoded;charset=utf-8',
+    },
+  });
+
+  const { data } = result;
+
+  const kakaoId = data.id;
+  const name = data.properties.nickname;
+  const email = data.kakao_account.email;
+  const age = data.kakao_account.age_range;
+  const gender = data.kakao_account.gender;
+
+  const age_range = Object.freeze({
+    '20~29': 1,
+    '30~39': 2,
+    '40~49': 3,
+    '50~59': 4,
+    '60~69': 5,
+    '70~79': 6,
+    '80~89': 7,
+  });
+
+  const gender_type = Object.freeze({
+    male: 1,
+    female: 2,
+  });
```

Comment:
```suggestion
  const ageRange = Object.freeze({
    '20~29': 1,
    '30~39': 2,
    '40~49': 3,
    '50~59': 4,
    '60~69': 5,
    '70~79': 6,
    '80~89': 7,
  });

  const genderType = Object.freeze({
    male: 1,
    female: 2,
  });
```
genderType이 여기에는 1과2만 있는데, 아래 코드에서는 4가 등장하네요?
필요한 값들은 전부 정의해 주시면 좋을 것 같습니다.